ui/package.json

```diff
@@ -5,8 +5,8 @@
     "ajv": "^8.12.0",
```

Comment:
We have 4 warnings to fix
<img width="776" alt="image" src="https://github.com/user-attachments/assets/115de120-a862-4921-872a-b473ee31f19d">